src/visualization/GraphRenderer.cc

```diff
@@ -123,27 +94,34 @@ void GraphRenderer::eventHandler(sf::Event event, sf::RenderWindow& window, vect
     }
 }
 
-void GraphRenderer::scoreHandler(sf::Event event, sf::RenderWindow& window) {
-    window.setView(scoreBar);
-    //example to include events
-    if (event.type == sf::Event::EventType::MouseButtonPressed) {
-        if (event.mouseButton.button == sf::Mouse::Middle) {
-            sf::RectangleShape temp;
-            temp.setPosition(sf::Vector2f(0, 0));
-            temp.setSize(scoreBar.getSize());
-            temp.setFillColor(sf::Color(128, 128, 255));
-            window.draw(temp);
-            example = temp;
-        }
-    }
+void GraphRenderer::display_score(sf::RenderWindow& window, const state& gameState) {
+    int x_pos = window.getView().getCenter().x - (window.getSize().x / 2);
+    int y_pos = window.getView().getCenter().y - (window.getSize().y / 2);
+
+    sf::RectangleShape rect(sf::Vector2f(window.getSize().x, 130));
+    rect.setPosition(x_pos, y_pos);
+  
+    sf::Font font;
+    if (!font.loadFromFile("Amiko-Regular.ttf"))
+        std::cout << "Can't find the font file" << std::endl;
+
+    std::string PlayerScore = "Your Score: " + std::to_string(gameState.score);
+    std::string AgentScore = "Computer Score: " + std::to_string(gameState.score); // TODO: need a score from an Agent    
```

Comment:
```suggestion
    std::string AgentScore = "Computer Score: " + "?"; // TODO: need a score from an Agent    
```

Reply:
To make it clear in future when this function has been implemented.

Reply:
This doesn't work. 
*error: invalid operands of types ‘const char [17]’ and ‘const char [3]’ to binary ‘operator+’
     std::string AgentScore = "Computer Score: " + "/?";*
 I change it to (std::string)"?"